src/features/todos.ts

```diff
@@ -1,9 +1,32 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type SetAddAction = {
+  type: 'todos/SET_ADD',
+  playload: Todo[],
```

Comment:
There seems to be a typo in your action payload. It should be 'payload' not 'playload'